spec/FooSpec.php

```diff
@@ -39,9 +39,9 @@
             $param = 'foo';
             $expected = $param.' processed';
 
-            Stub::on($this->dependency)->method('process')
-                                       ->with($param)
-                                       ->andReturn($expected);
+            allow($this->dependency)->method('process')
```

Comment:
Hi, the API is `allow(...)->toReceive(...)` similar to `expect(...)->toReceive(...)`